### DISCUSSION TITLE
Filecoin Devnets

### DISCUSSION THREAD

**Discussion by jennijuju** [filecoin-project/community#303] [General]
Use discussion to share ideas, use cases, requirements for future Filecoin Community Devnets!

Announcement see https://github.com/filecoin-project/community/discussions/74#discussioncomment-1348469.

**Comment by mishmosh** [comment]
**Background:** After thoughtful reviewing the usage & ongoing issues with nerpanet with the DMob team, we have decided to spin the nerpanet down. The announcement was made in #fil-net-nerpa-discuss Slack channel, and in the Network Announcements thread: https://github.com/filecoin-project/community/discussions/74#discussioncomment-1348469

The team is writing a “lessons learned” doc, but highlights :
- the tools & configs for spawning & maintaining networks were designed for mainnet scale, and don’t work nearly as well at small devnet scale 
 - this was a single network trying to serve too many varied demands & needs (test sector sizes, market volume, etc.).

That said, these problems and your community involvement have pointed us to a new approach for devnets. Next, the NetOps team at Protocol Labs is building new and easy to use tools for spinning up devnets, and will be releasing them along with a runbook to the community for your own use. We can’t wait to get these tools in your hands and see the devnets you create for different use cases!

If you have any requests/ideas, please chime in here!

**Comment by rickiey** [comment]
我希望新的测试网能够提供快速证明的小型扇区，我们在测试seal无关的部分代码时，总是把时间花费在seal上，并且它还需要昂贵的硬件（增加门槛？），所以希望能够提供小型扇区来降低测试时间和硬件成本。
google translate: I hope that the new testnet can provide small sectors that can be quickly proved. We always spend time on sealing when testing parts of code that are irrelevant to the seal, and it also requires expensive hardware (increases the threshold?), so I hope to be able to Provide small sectors to reduce test time and hardware cost

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
as by now we provide a public 512MiB testnet. beta testing right now - stable 1.1.2022

join the slack and the chan #fil-net-community-discuss for details

**Comment by gdragonHH** [comment]
硬件算力出海，专业技术团队 独立节点，欢迎大家加V了解：Hmcxxxxx

**Comment by sianliu** [comment]
@NetOps team This is a bit off topic. Let me apologise in advance. I saw on the Protocol Labs career page that you're looking for a Fil-Infra DevOps/SRE Engineer. I've worked in tech for over 10 years building infra. This past year, I've been running a public good DeFi project on Ethereum. We weren't able to raise. I'm looking to move back into DevOps. I started my web3 journey 2 years ago. And before that I was in DevOps. But the listing says you guys need 4 and I only have 2. :-> Should I still apply? Thanks in advance.
